src/components/ExpandableBlockquote.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { RefObject } from "react";
+import Chevron from "nav-datovelger/lib/elementer/ChevronSvg";
+
+const texts = {
+  readMore: "vis mer",
+  readLess: "vis mindre",
+};
+
+interface ExpandableBlockquoteProps {
+  body: string;
+  className?: string;
+}
+
+export const ExpandableBlockquote = ({
+  body,
+  className,
+}: ExpandableBlockquoteProps) => {
+  const [viewAll, setViewAll] = React.useState<boolean>(false);
+  const ref = React.useRef() as RefObject<any>;
+  const isOverflow = useIsOverflow(ref, () => {
+    /*noop*/
+  });
+  const showButton = isOverflow || viewAll;
+  const panelHeight = viewAll ? "h-auto" : "h-24";
+
+  return (
+    <div className="grid grid-cols-1 place-items-center">
+      <blockquote
+        className={`${className} ${panelHeight} mb-2 overflow-hidden w-full`}
+        ref={ref}
+      >
+        {body}
+      </blockquote>
+      {showButton && (
+        <ExpandButton setOpen={setViewAll} isOverflow={!!isOverflow} />
+      )}
+    </div>
+  );
+};
+
+const ExpandButton = ({ setOpen, isOverflow }: ExpandButtonProps) => {
+  const nextOpenValue = isOverflow;
```

Comment:
Vet ikke om jeg skjønte helt `nextOpenValue`-navnet 🤔 For de to ternary sjekkene på linje 43 og 44 synes jeg `isOverflow` leste bedre, men det er kanskje for linje 49 det skal bli lettere å forstå?

Reply:
Ja, det ble ikke så tydelig nå! Det jeg tenkte var en ulempe er at isOverflow nå er både en generell boolean for "det er nok tekst til at vi kan åpne/lukke" og "er det overflow akkurat nå, eller er den åpen". 🤔